platforms/software/dependency-management/src/main/java/org/gradle/internal/component/resolution/failure/describer/AmbiguousArtifactTransformsFailureDescriber.java

```diff
@@ -35,13 +35,18 @@
  * A {@link ResolutionFailureDescriber} that describes an {@link AmbiguousArtifactTransformsFailure}.
  */
 public abstract class AmbiguousArtifactTransformsFailureDescriber extends AbstractResolutionFailureDescriber<AmbiguousArtifactTransformsFailure> {
+    private static final String ARTIFACT_TRANSFORMS_REPORT_SUGGESTION = "Run the :artifactTransforms report to see the available artifact transforms.";
```

Comment:
This will be the wrong task path if the failure occurs in a subproject or another build.

Reply:
It's true, and I looked briefly at getting the project information and storing it with the registration, but that didn't look easily possible.  I figured these suggestions have always been heuristics and don't need to be 100% correct all of the time.